internal/scorecard/plugins/olm_tests.go

```diff
@@ -176,6 +181,42 @@ func NewOLMTestSuite(conf OLMTestConfig) *schelpers.TestSuite {
 
 // Test Implentations
 
+// Run - implements Test interface
+func (t *BundleValidationTest) Run(ctx context.Context) *schelpers.TestResult {
+	res := &schelpers.TestResult{Test: t, MaximumPoints: 1}
+
+	if t.OLMTestConfig.Bundle == "" {
+		res.Errors = append(res.Errors, errors.New("add a OLM 'bundle' directory which is required for this test"))
+		return res
```

Comment:
The msg error here shows the suggestion for the fix. However, as it is an error, the msg should not be something as `unable to find the OLM 'bundle' directory which is required for this test`

Reply:
I'm updated that error message to what you recommend to make it clearer.